### PR TITLE
Doc update recommendations

### PR DESCRIPTION
**Pull request by thgarrett27**
Minor updates to intro and introducing_rest topics. Trying to simplify language and remove anything unnecessary. Also focused on removing as much of the future tense as possible.

### CODE REVIEW

File: docs/Overview/introducing-rest.md [hunk 1]
```diff
@@ -4,51 +4,51 @@ sidebar_position: 1
...
+This chapter introduces you to DreamFactory, an automated REST API generation, integration, and management platform. You can use DreamFactory to generate REST APIs for hundreds of data sources, including MySQL and Microsoft SQL Server databases, file systems including Amazon S3, and e-mail delivery providers like Mandrill. You can also integrate third-party APIs, including all of the services mentioned above. This opens up a whole new world of possibilities in terms of building sophisticated workflows. But before we jump into this introduction, some readers might be wondering what a REST API is in the first place, let alone why so many organizations rely on REST for their API implementations.
 
 ## Introducing REST
 
-If you were to design an ideal solution for passing data between computers ("computers" being an umbrella term used to represent servers, laptops, mobile phone, and any other Internet-connected device), what would it look like?
+If you were to design an ideal solution for passing data between computers ("computers" here refers to any servers, laptops, mobile phones, ory other Internet-connected device), what would it look like?
```
Comment (thekevinm) [inline docs/Overview/introducing-rest.md:15]: typo

File: docs/Overview/introducing-rest.md [hunk 2]
```diff
@@ -156,59 +156,59 @@ Suppose the `employees` table includes attributes such as `first_name`, `last_na
...
 
     `DELETE /api/v2/employees/42`
 
 ## Introducing DreamFactory
 
-In light of everything we've discussed thus far with regards to implementing a REST API, the idea of implementing one yourself probably sounds pretty daunting. It should, because it is. In doing so, not only would you be responsible for building out the logic required to process the request methods and URLs, but you'd also be on the hook for integrating authentication and authorization, generating and maintaining documentation, and figuring out how to sanely generate working APIs for any number of third-party data sources.
+With all that we've covered in this topic, the thought of implementing a REST API on your own probably sounds quite daunting. It should, because it is. Nnot only would you be responsible for building out the logic required to process the request methods and URLs, but you'd also be on the hook for integrating authentication and authorization, generating and maintaining documentation, and figuring out how to sanely generate working APIs for any number of third-party data sources. It's a handful!
```
Comment (thekevinm) [inline docs/Overview/introducing-rest.md:165]: typo

File: docs/Overview/introducing-rest.md (hunk 1 above)
Comment (thgarrett27) [reply docs/Overview/introducing-rest.md]: Corrected.

File: docs/Overview/introducing-rest.md (hunk 2 above)
Comment (thgarrett27) [reply docs/Overview/introducing-rest.md]: Corrected.

**Pull request merged**